source/features/tags-on-commits-list.tsx

```diff
@@ -1,17 +1,18 @@
 import React from 'dom-chef';
 import cache from 'webext-storage-cache/legacy.js';
-import {$$} from 'select-dom';
 import {$} from 'select-dom/strict.js';
 
 import TagIcon from 'octicons-plain-react/Tag';
 import * as pageDetect from 'github-url-detection';
+import batchedFunction from 'batched-function';
 
 import features from '../feature-manager.js';
 import api from '../github-helpers/api.js';
```

Comment:
The api/query setup is complex here, because it relies on the "last commit on the page", which now is not what we expect. 

This is why I never ported this feature to the observer. I wonder if we should just fix the original bug via polling instead of changing this whole file. I never reviewed/touched this feature.

I'd probably be more comfortable with a rewrite from scratch than trying to iterate on it, but since this is a GHE-only but maybe we should just use polling + `$$` somehow. Any ideas?